tpp-app/src/home/pages/LogMultipleDatesScreen.js

```diff
@@ -31,14 +33,22 @@ const DayComponent = ({props}) => {
     )
 }
 
-export const Calendar = ({ navigation, setSelectedDates, markedDates }) => {
+var joinedDate = ""; 
+GETJoinedDate().then(res => { joinedDate = res })
```

Comment:
same here, please move inside Calendar or LogMultipleDatesScreen and then pass joinedDate to Calendar